src/cli/commands/install.js

```diff
@@ -134,6 +134,8 @@ export class Install {
     this.config = config;
     this.flags = normalizeFlags(config, flags);
 
+    this.config.production = this.flags.production;
```

Comment:
We can move that to `config.init` in `cli/index.js` and move the `--production` flag from `install.js` to `cli/index.js`.